tests/test_buffer_traditional.py

```diff
@@ -78,16 +78,15 @@ def get_pg_name_map(self):
 
     @pytest.fixture
     def setup_teardown_test(self, dvs):
-        try:
-            self.setup_db(dvs)
-            self.set_port_qos_table(self.INTF, '2,3,4,6')
-            pg_name_map = self.get_pg_name_map()
-            yield pg_name_map
-        finally:
-            self.teardown()
+        self.setup_db(dvs)
+        self.set_port_qos_table(self.INTF, '2,3,4,6')
+        time.sleep(2)
+
+        yield
```

Comment:
Now that nothing is returned from yield, you man also want to update `test_buffer_pg_update` accordingly

Reply:
Done